mmv1/products/secretmanagerregional/RegionalSecret.yaml

```diff
@@ -0,0 +1,175 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: RegionalSecret
+self_link: projects/{{project}}/locations/{{region}}/secrets/{{secret_id}}
+base_url: projects/{{project}}/locations/{{region}}/secrets
+create_url: projects/{{project}}/locations/{{region}}/secrets?secretId={{secret_id}}
+update_verb: :PATCH
+update_mask: true
```

Comment:
This resource will need a [handwritten update test](https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test) that changes the values of all fields that can be updated in-place.